File: unity/Assets/Scripts/Multiplayer/Players/PlayerUnbanResponse.cs
using Michitai.Multiplayer.Errors;
using System;
using UnityEngine;

namespace Michitai.Multiplayer.Players
{
    /// <summary>
    /// Response returned when a player is successfully unbanned.
    /// </summary>
    [Serializable]
    public class PlayerUnbanResponse : ApiResponse<ECommonError>
    {
        /// <summary>
        /// Confirmation message for the unban.
        /// </summary>
        public string message;

        /// <summary>
        /// The ID of the player who was unbanned.
        /// </summary>
        public int player_id;
    }
}

File: java/src/main/java/com/michitai/multiplayer/players/PlayerUnbanResponse.java
package com.michitai.multiplayer.players;

import com.fasterxml.jackson.annotation.JsonProperty;
import com.michitai.multiplayer.ApiResponse;

/**
 * Response confirming a player was unbanned.
 */
public class PlayerUnbanResponse extends ApiResponse {
    @JsonProperty("message")
    private String message;

    @JsonProperty("player_id")
    private int playerId;

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }

    public int getPlayerId() {
        return playerId;
    }

    public void setPlayerId(int playerId) {
        this.playerId = playerId;
    }
}
